sources/zendesk_pipeline.py

```diff
@@ -52,7 +59,7 @@ def incremental_load_all_start_time() -> Any:
     # Choosing starting point for incremental load - optional, the default is the last load time. If no last load time
     # the start time will be the 1st day of the millennium
     # start time needs to be a pendulum datetime object
-    start_time = pendulum.DateTime(year=2023, month=1, day=1)
+    start_time = pendulum.DateTime(year=2023, month=1, day=1).in_timezone("UTC")
```

Comment:
uh, is it really necessary? pendulum imported from dlt module is patched to always use UTC

Reply:
I thought so too,  parsed tz naive dates and such all assume UTC, but does not seem to apply to the `DateTime` constructor:

```python
In [6]: pendulum.parse("2022-03-03T01:02:03")
Out[6]: DateTime(2022, 3, 3, 1, 2, 3, tzinfo=Timezone('UTC'))

In [7]: pendulum.DateTime(2023, 1, 1, 1, 1, 1)
Out[7]: DateTime(2023, 1, 1, 1, 1, 1)
```

Not sure if there's some simple way to prevent naive dates entirely. Maybe we can patch the `DateTime` class